--- a/cctools-port/cctools/ld64/src/ld/PlatformSupport.cpp	2021-01-18 15:59:18.176513899 -0600
+++ b/cctools-port/cctools/ld64/src/ld/PlatformSupport.cpp	2021-01-18 15:55:44.610422388 -0600
@@ -177,8 +177,8 @@
                             break;
                     case PlatEnforce::warning: {
                         if ( !warned ) {
-                            warning("building for %s, but linking in %s file (%s) built for %s",
-                                    to_str().c_str(),  dylibType.c_str(), targetPath.c_str(), dylibPlatforms.to_str().c_str());
+                            // warning("building for %s, but linking in %s file (%s) built for %s",
+                            //         to_str().c_str(),  dylibType.c_str(), targetPath.c_str(), dylibPlatforms.to_str().c_str());
                             warned = true;
                         }
                     } break;
